server/routes/users.js

```diff
@@ -8,42 +9,57 @@ import config from '../lib/config';
 import { verifyUserAccess } from '../lib/middlewares';
 import removeGuardian from '../lib/removeGuardian';
 
+function executeWriteHook(req, res, scriptManager, userFields) {
+  return req.auth0.users.get({ id: req.params.id })
+    .then((user) => {
+      if (!user) {
+        res.status(404);
+        throw new Error('User not found');
+      }
+      const context = {
+        method: 'update',
+        request: {
+          user: req.user,
+          originalUser: user
+        },
+        payload: req.body,
+        userFields
+      };
+      return context;
+    })
+    .then(context => scriptManager.execute('create', context));
```

Comment:
There is not `create` script anymore, but instead a `write` script should that not be reflected in this code as well? If this is for backwards compatibility, what's the strategy to change this at a later stage?

Reply:
There is a way to create an update hook.  Perhaps we need to create an update hook for this one...

Reply:
It could rename the create hook to the write hook.